samples/Tutorial/CarryForward.fs

```diff
@@ -0,0 +1,116 @@
+namespace Tutorial.ClosingBooks
+
+open FSharp.UMX
+
+/// Buffers events accumulated from a series of decisions while evolving the presented `state` to reflect said proposed `Events`
+type private Accumulator(originState, fold) =
+    let mutable state = originState
+    let pendingEvents = ResizeArray()
```

Comment:
`pendingEvents` is never added to?

Reply:
yikes, cut/paste accident, nice catch thanks!

Reply:
(fixes now, see `apply`)